src/System.Private.ServiceModel/tools/PRService/web.config

```diff
@@ -8,7 +8,21 @@
 <configuration>
   <system.web>
     <compilation debug="false" />
-    <httpRuntime/>
+    <!-- some git syncs may take a long time, so upping the executionTimeout to help with this -->
+    <httpRuntime executionTimeout="300" />
   </system.web>
+  <appSettings>
+    <!--
+      These appSettings provide the default environment under which we execute
+      These should be changed to reflect what is correct on the local system, or 
+      specified as a system-wide environment variable. 
+      
+      Environment variables take precedence over what is specified here
+    -->
+    <add key="GitExecutablePath" value="C:\Program Files\Git\cmd\git.exe"/>
+    <add key="GitRepositoriesBasePath" value="C:\git"/>
+    <add key="GitExecutionTimeout" value="00:01:00"/>
```

Comment:
Reviewers: i changed the total execution timeout to be 300s, but each git command should only take 60s or less (we aren't building anything). 

We want the timeout to happen on the command rather than relying on the overall ASP.NET timeout because if we hit the ASP.NET timeout, all output is via context.Response will be suppressed. The way the code is written, a git command timing out will spew out an error message and tell you which git command caused the problem